src/gql/queries/get-task.ts

```diff
@@ -69,6 +69,7 @@ export const GET_TASK = gql`
       annotation {
         taskId
         taskExecution
+        userCanModify
```

Comment:
Why is this needed here if you don't use this value anywhere in your PR?

Reply:
One of the components in spruce depends on having this type. So I had to include it to appease typescript

Reply:
So you remember which component?